### ISSUE TITLE
Table label to state which sort function is applied

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3086]
Add an audio/visual tag to show which sorting option has been used in the table.

 “When Jaws reads out the column headers it announces that you can change the information in the table. I was unaware that clicking the filters would perform a similar action.” Ensure that the column header links inform screen reader users that the filter has been successful when selected to help screen reader users identify their purpose. The state of the column should inform users without the need to move focus away from and back onto the link.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [ ] Windows: IE11
- [ ] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
Potentially @dougmet to help with this one.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Table label to state which sort function is applied](https://trello.com/c/2e66QTlh/610-table-label-to-state-which-sort-function-is-applied)

**Comment by Norric1Admin** [comment]
Changing this to `AAA` as it still requires a fix medium-term, but is no longer needed for `AA` accreditation as long as we provide the alternative below.
___
I've spoken to DAC and they are willing to give a little. The table feature isn't accessible, but the downloadable Excel file is. As long as we do the following, the 'AA' rating will be confirmed.

This line needs to be added to the catch-all text of every published statistical indicator (might be best done through Python?).

TEMPLATE: "Select the following link to download [ indicator : title [FILE SIZE]]((https://sustainabledevelopment-uk.github.io/sdg-data/data permalink .csv) in an accessible format"

EXAMPLE: "Select the following link to download [1.2.1: Proportion of population living below the national poverty line, by sex and age [15KB]](https://sustainabledevelopment-uk.github.io/sdg-data/data/1-2-1.csv) in an accessible format"

**Comment by Norric1Admin** [comment]
This is being tested on https://sdg.mango-solutions.com/download-test-deploy/1-1-1/

We still need to merge all indicators into the branch, but the test passed at the required level.

**Comment by LucyGwilliamAdmin** [comment]
@Norric1Admin I started this, but think it failed and needed to do some fixes so that all indicators can be merged. Forgot to do it as was out of office.

**Comment by LucyGwilliamAdmin** [comment]
Actually, looks like it passed

**Comment by AnnCorp** [comment]
Resource focussed on achieving AA, so won't be actioning AAA fixes currently.
